### DISCUSSION TITLE
I cannot use copilot.

### DISCUSSION THREAD

**Discussion by Nurxat21** [community/community#113420] [GitHub Education]
### Hi everyone,

You see! I have verified as a student. But, I cannot use the copilot. 
![WechatIMG125](https://github.com/community/community/assets/75790366/6d72a479-e7c7-466e-aa79-3a69c5ea9fd6)

**Comment by SebastienXia** [comment]
## Hello, @Nurxat21 

I think you went a little too quickly, you were verified so approved during your student process but you remain without the benefits of the pack. (you do not have the Pro in your profile)

> Let me explain why? Quite simply because you have to wait between 5 - 7 days after being approved, and you will receive an email saying that you have access to the benefit of the pack.

![Capture](https://github.com/community/community/assets/105004912/eebeb57c-033e-4451-ac84-274ed0f0cc0e)
Normally your approval isn't purple but green, it's mean than you don't have yet pack benefits.

`[close the thread and mark as answered if this helped you resolve your issue]`
